client/scss/overrides/_utilities.legacy.scss

```diff
@@ -1,3 +1,8 @@
+.u-hidden {
+  // stylelint-disable-next-line declaration-no-important
+  display: none !important;
+}
```

Comment:
All other `hidden` utilities have been switched to using Tailwind, as they were only used in very few spots after the legacy sidebar removal.